### PR TITLE
TRIDENT-17567 - Doc for Insight Trainer

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a new article for how to use the new "Cloud SIEM - Insight Trainer" dashboard:
https://help.sumologic.com/docs/cse/rules/insight-trainer

It also adds the Insight Trainer dashboard to the list of dashboards for the SIEM app:
https://help.sumologic.com/docs/integrations/sumo-apps/cse/#view-app-dashboards

Issue number: 
[TRIDENT-17567](https://jasklabs.atlassian.net/browse/TRIDENT-17567)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
Pull request is in draft.

**Comment by jpipkin1** [comment]
Sourabh Bhosale reviewed the [Google Doc draf](https://docs.google.com/document/d/1_uymJA9jeO-n97YDKNIlzgCKKaVgYzS2jyWCAuMFLow/edit?userstoinvite=nrama@sumologic.com#)t and gave his approval via Slack.

File: docs/cse/rules/insight-trainer.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  - rule tuning
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+[Cloud SIEM - Insight Trainer](/docs/integrations/sumo-apps/cse#insight-trainer) is a dashboard in the Enterprise Audit - Cloud SIEM app. Insight Trainer offers suggestions for making adjustments to rules, such as writing rule tuning expressions and changing severities. Implementing the recommendations causes rules to be more effective at creating high-fidelity Signals, resulting in generation of more meaningful Insights. 
+
+## About Insight Trainer
+
+When you resolve Insights, you give them [Insight resolutions](/docs/cse/administration/manage-custom-insight-resolutions/) such as:
```
Comment (kimsauce) [inline docs/cse/rules/insight-trainer.md:18]: Should this be "you're given"?

File: docs/cse/rules/insight-trainer.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+[Cloud SIEM - Insight Trainer](/docs/integrations/sumo-apps/cse#insight-trainer) is a dashboard in the Enterprise Audit - Cloud SIEM app. Insight Trainer offers suggestions for making adjustments to rules, such as writing rule tuning expressions and changing severities. Implementing the recommendations causes rules to be more effective at creating high-fidelity Signals, resulting in generation of more meaningful Insights. 
+
+## About Insight Trainer
+
+When you resolve Insights, you give them [Insight resolutions](/docs/cse/administration/manage-custom-insight-resolutions/) such as: 
+
+* False Positive: A false alarm, possibly due to an error in the detection logic of the rule or inapplicability to your environment.
+* No Action: A valid detection but no action is necessary due to effective containment measures. Rules participating in repeated No Action Insights may also be good tuning expression candidates.
+* Resolved: A valid detection where investigation was necessary.
```
Comment (kimsauce) [inline docs/cse/rules/insight-trainer.md:22]:
```suggestion
* **False Positive**. A false alarm, possibly due to an error in the detection logic of the rule or inapplicability to your environment.
* **No Action**. A valid detection, but no action is necessary due to effective containment measures. Rules participating in repeated No Action Insights may also be good tuning expression candidates.
* **Resolved:**. A valid detection where investigation was necessary.
```

File: docs/cse/rules/insight-trainer.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+In general, fewer Insights are generated if you decrease severities on rules, and more Insights are generated if you increase severities on rules. If more Insights are generated because you increase severity, the incrementally changed Insights no longer have a resolution label (are “unlabeled”). 
+
+## Suggested workflow
+
+Following is the suggested workflow to use the Insight Trainer dashboard:
+1. Review severity recommendations once a week.
+1. Look at the rules with the highest tunability scores and assess the dominant Entities to write [tuning expressions](/docs/cse/rules/rule-tuning-expressions) for (like users and IP addresses).
```
Comment (kimsauce) [inline docs/cse/rules/insight-trainer.md:93]:
```suggestion
1. Look at the rules with the highest tunability scores and assess the dominant Entities to write [rule tuning expressions](/docs/cse/rules/rule-tuning-expressions) for (like users and IP addresses).
```

**Review by kimsauce — COMMENTED** [review]
had a few minor Qs

File: docs/cse/rules/insight-trainer.md (hunk 1 above)
Comment (jpipkin1) [reply docs/cse/rules/insight-trainer.md:18]: It's good as-is. The customer actually gives a resolution to an Insight when they close it.

**Comment by jpipkin1** [comment]
Thanks for the quick review, @kimsauce! I took two of your suggestions and left a third as-is.

**Comment by jpipkin1** [comment]
@kimsauce, could you give it one more look?

**Comment by jpipkin1** [comment]
@kimsauce, they want to release the feature now. Could you approve the PR so I can publish the doc?

**Review by kimsauce — APPROVED** [review]

**Pull request merged**